--- libxklavier-0.95/configure.in.wiget	2003-09-20 01:36:53.000000000 +0200
+++ libxklavier-0.95/configure.in	2003-11-16 18:27:09.000000000 +0100
@@ -56,9 +56,9 @@
              xkb_base="$withval", 
              xkb_base="/usr/X11R6/lib/X11/xkb" )
 			 
-if ! test -d $xkb_base; then
-        AC_MSG_ERROR([The path $xkb_base does not denote the directory])
-fi
+#if ! test -d $xkb_base; then
+#        AC_MSG_ERROR([The path $xkb_base does not denote the directory])
+#fi
 
 AC_DEFINE_UNQUOTED(XKB_BASE,"${xkb_base}",Base for XKB configuration)
 
