diff -aurN libxklavier-2.91.orig/libxklavier/Makefile.am libxklavier-2.91/libxklavier/Makefile.am
--- libxklavier-2.91.orig/libxklavier/Makefile.am	2006-03-12 04:49:22.000000000 +0100
+++ libxklavier-2.91/libxklavier/Makefile.am	2006-06-13 18:10:52.236418968 +0200
@@ -24,7 +24,7 @@
 xkl_engine_marshal.c: xkl_engine_marshal.h
 	$(GLIB_GENMARSHAL) --prefix=xkl_engine marshal.list --body > xkl_engine_marshal.c
 
-AM_CFLAGS=-Wall -Werror -DDATA_DIR=\"$(datadir)/$(PACKAGE)\" \
+AM_CFLAGS=-Wall -DDATA_DIR=\"$(datadir)/$(PACKAGE)\" \
   -I. -I$(includedir) $(X_CFLAGS) -I$(top_srcdir) \
   $(XML_CFLAGS) $(GLIB_CFLAGS) \
   $(XKB_HEADERS_PRESENT_CFLAG) \
