--- libxklavier-1.04/libxklavier/xklavier_config_i18n.c.orig	2003-09-20 01:37:05.000000000 +0200
+++ libxklavier-1.04/libxklavier/xklavier_config_i18n.c	2005-03-02 11:39:04.000000000 +0100
@@ -106,7 +106,7 @@
   int bytesWritten;
 
   iconv_t converter;
-  static char converted[XKL_MAX_CI_DESC_LENGTH];
+  char converted[XKL_MAX_CI_DESC_LENGTH];
   char *convertedStart = converted;
   char *utfStart = ( char * ) utf8string;
   int clen = XKL_MAX_CI_DESC_LENGTH - 1;
@@ -148,7 +148,7 @@
 
   iconv_close( converter );
 
-  return converted;
+  return strdup( converted );
 }
 
 /*
